### PR TITLE
Add models arguments to cli

### PR DESCRIPTION
**Pull request by frankfliu**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: frontend/server/src/main/java/com/amazonaws/ml/mms/ModelServer.java [hunk 1]
```diff
@@ -111,24 +129,25 @@ public void initModelStore() throws InvalidModelException, WorkerInitializationE
...
-                        if (!model.endsWith(".mar")) {
-                            modelFile = new File(modelStore, model + ".mar");
-                        }
+                    String[] pair = model.split("=", 2);
+                    String modelName = null;
+                    String url;
+                    if (pair.length == 1) {
+                        url = pair[0];
+                    } else {
+                        modelName = pair[0];
```
Comment (vrakesh) [inline frontend/server/src/main/java/com/amazonaws/ml/mms/ModelServer.java]: Should we validating urls here? in this file?

File: mms/model_server.py [hunk 2]
```diff
@@ -2,31 +2,47 @@
...
 
 def start():
     """
     This is the entry point for model server
     :return:
     """
     args = ArgParser.mms_parser().parse_args()
-    if args.start is True:
-        os.environ['MODEL_SERVER_HOME'] = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    pid_file = tempfile.gettempdir() + "/.model_server.pid"
```
Comment (vrakesh) [inline mms/model_server.py]: os.path.join() should be used instead

File: frontend/server/src/main/java/com/amazonaws/ml/mms/ModelServer.java (hunk 1 above)
Comment (frankfliu) [reply frontend/server/src/main/java/com/amazonaws/ml/mms/ModelServer.java]: This is handled in ModelArchive

File: mms/model_server.py (hunk 2 above)
Comment (frankfliu) [reply mms/model_server.py]: Will change in following PR

**Pull request merged**